### PR TITLE
Freeze typescript at 3.7.2 for stability.

### PR DESCRIPTION
**Pull request by shaper**

### CODE REVIEW

**Comment by shans** [comment]
Do we want to do this, or do we want to try and fix the issues with 3.7.2 first?

**Comment by Cypher1** [comment]
I think we should upgrade, but at our own pace. It seems likely that the upgrade has been part of the recent build breakage.

**Comment by shaper** [comment]
Updated to freeze at 3.7.2.

File: src/tools/vscode-language-client/package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "devDependencies": {
         "mocha": "^6.1.4",
         "tslint": "^5.14.0",
-        "typescript": "^3.0.0",
+        "typescript": "3.6.3",
```
Comment (csilvestrini) [inline src/tools/vscode-language-client/package.json]: Why not 3.7.2 here?

**Pull request closed without merging**